packages/adapter-d1/README.md

```diff
@@ -1,3 +1,94 @@
 # Prisma driver adapter for Cloudflare D1
 
-**INTERNAL PACKAGE, DO NOT USE**
+Prisma driver adapter for [Cloudflare D1](https://developers.cloudflare.com/d1/).
+
+<!-- TODO Refer to the [announcement blog post](https://prisma.io/cloudflare-d1) and our [docs](https://www.prisma.io/docs/guides/database/cloudflare-d1) for more details. -->
+
+> **Note**: Support for Cloudflare D1 is available in [Early Access](https://www.prisma.io/docs/about/prisma/releases#early-access) from Prisma versions [TODO](https://github.com/prisma/prisma/releases/tag/TODO) and later.
+
+## Getting started
+
+To get started, enable the `driverAdapters` Preview feature flag in your Prisma schema:
+
+```prisma
+// schema.prisma
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["driverAdapters"]
+}
+
+datasource db {
+  provider = "sqlite"
+  url      = "file:./dev.db"
+}
+```
+
+Generate Prisma Client:
+
+```sh
+npx prisma generate
+```
+
+Install the Prisma adapter for Cloudflare D1 and Cloudflare's workers types packages:
+
+```sh
+npm install @prisma/adapter-d1
+npm install @cloudflare/workers-types
+```
```

Comment:
Actually, we should probably also include:
```
npm install wrangler
```

Reply:
Why? Is that required to run Prisma Client? Or just some other workflow?

Reply:
It's required for actually then executing sql against the database right now as we don't explicitly handle migrations. But maybe that's less relevant for now if we're just focused on client